content/docs/user-guide/basic-concepts/dvc-project.md

```diff
@@ -16,5 +16,5 @@ match:
 Initialized by running `dvc init` in the **workspace** (typically a Git
 repository). It will contain the
 [`.dvc/` directory](/doc/user-guide/dvc-files-and-directories) and
-[DVC-files](/doc/user-guide/dvc-file-format) created with commands such as
+[`.dvc` files](/doc/user-guide/dvc-file-format) created with commands such as
```

Comment:
Should read:

> The .dvc/ directory, as well as dvc.yaml and .dvc files created...

With corresponding files (and formatting), please

Reply:
Should I link all of these `.dvc` file and dvc.yaml to /dvc-file-format?

Reply:
Not all, just the first ones in each section of a doc. In this case I think yes, link both.

Reply:
Moved to #1408